airbyte-cdk/bulk/core/base/src/main/kotlin/io/airbyte/cdk/output/OutputConsumer.kt

```diff
@@ -33,11 +33,18 @@ import java.util.function.Consumer
 
 /** Emits the [AirbyteMessage] instances produced by the connector. */
 @DefaultImplementation(StdoutOutputConsumer::class)
-interface OutputConsumer : Consumer<AirbyteMessage>, AutoCloseable {
-    val emittedAt: Instant
+abstract class OutputConsumer(private val clock: Clock) : Consumer<AirbyteMessage>, AutoCloseable {
+    /**
+     * The constant emittedAt timestamp we use for record timestamps.
+     *
+     * TODO: use the correct emittedAt time for each record. Ryan: not changing this now as it could
```

Comment:
very low (60%) confidence: this timestamp being constant in records is intentional, to reflect that sources snapshot the source DB at a point in time. `emitted_at` is interpreted by customers as "when was the data extracted from the source"

but would love to get confirmation from the sources side on that.